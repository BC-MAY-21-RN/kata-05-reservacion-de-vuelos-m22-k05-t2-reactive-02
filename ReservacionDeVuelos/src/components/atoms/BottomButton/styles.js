import {StyleSheet, Dimensions} from 'react-native';

const styles = StyleSheet.create({
  containerButton: {
    backgroundColor: '#5C6DF8',
    borderRadius: 10,
    height: 45,
    width: Dimensions.get('screen').width - 60,
    position: 'absolute',
    alignItems: 'center',
    alignContent: 'center',
    alignSelf: 'center',
    bottom: 45,
    justifyContent: 'center',
  },
  textButton: {color: '#ffffff', fontSize: 18},
});

export default styles;
