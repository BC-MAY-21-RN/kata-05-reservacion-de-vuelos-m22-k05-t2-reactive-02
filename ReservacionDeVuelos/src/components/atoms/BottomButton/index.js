import React from 'react';
import {Text, TouchableOpacity} from 'react-native';
import styles from './styles';

export default function BottomButton({navigation, values}) {
  return (
    <TouchableOpacity
      style={styles.containerButton}
      onPress={() => navigation.navigate('FinishScreen', {values})}>
      <Text style={styles.textButton}>Next</Text>
    </TouchableOpacity>
  );
}
