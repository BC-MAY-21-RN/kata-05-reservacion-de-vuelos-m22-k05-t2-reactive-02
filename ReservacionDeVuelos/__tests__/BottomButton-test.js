import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import BottomButton from '../src/components/atoms/BottomButton';

describe('Tests BottomButton component', () => {
  it('BottomButton snapshot', () => {
    const renderBottomButton = renderer
      .create(
        <BottomButton
          active={false}
          navigation={false}
          screenName={''}
          text={''}
          values={''}
        />,
      )
      .toJSON();
    expect(renderBottomButton).toMatchSnapshot();
  });
});
