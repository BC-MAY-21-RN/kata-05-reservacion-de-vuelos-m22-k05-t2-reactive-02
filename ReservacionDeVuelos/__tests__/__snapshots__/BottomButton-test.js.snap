// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests BottomButton component BottomButton snapshot 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignContent": "center",
      "alignItems": "center",
      "alignSelf": "center",
      "backgroundColor": "#5C6DF8",
      "borderRadius": 10,
      "bottom": 45,
      "height": 45,
      "justifyContent": "center",
      "opacity": 1,
      "position": "absolute",
      "width": 690,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#ffffff",
        "fontSize": 18,
      }
    }
  >
    Next
  </Text>
</View>
`;
